CONTRIBUTING.md

```diff
@@ -411,24 +450,22 @@ $ git tag -a -m "Release X.Y.Z" X.Y.Z
 $ git push mongodb --tags
 ```
 
-### Release PECL package
+Pushing the new tag will start the packaging process which provides the PECL 
+and Windows packages that should be attached to the release.
+
+#### Release PECL package
 
 The PECL package may be published via the
 [Release Upload](https://pecl.php.net/release-upload.php) form. You will have
 one chance to confirm the package information after uploading.
 
-### Update version info back to dev
+#### Update version info back to dev
 
 After tagging, the version and stability constants in `phongo_version.h` should
-be updated back to development status.
+be updated back to development status:
 
-```diff
--#define PHP_MONGODB_VERSION "1.1.8"
--#define PHP_MONGODB_STABILITY "stable"
--#define PHP_MONGODB_VERSION_DESC 1,1,8,1
-+#define PHP_MONGODB_VERSION "1.1.9-dev"
-+#define PHP_MONGODB_STABILITY "devel"
-+#define PHP_MONGODB_VERSION_DESC 1,1,9,0
+```shell
+$ ./bin/update-release-version.php patch
```

Comment:
Should you solely demonstrating `update-release-version.php patch` here? I think there are cases where `minor` may also be appropriate, especially since you removed the "After releasing a new minor version" section.

This content was originally under [Releasing](https://github.com/mongodb/mongo-php-driver/blob/1.18.1/CONTRIBUTING.md#releasing) and presented instructions common to both minor _and_ patch releases.

If this _should_ demonstrate `update-release-version.php patch`, then I think we're missing some other section that discusses when to use `update-release-version.php minor` (after cutting a new branch).

Reply:
I assumed that the "Branch Management" section is always done, regardless of the release process. Essentially, before releasing a new minor version we create the release branch for that minor release, then update the default branch to the next minor release. Only after that does the release process (either automated or manual) start, always working from the version branch.